BLAS/SRC/CMakeLists.txt

```diff
@@ -38,12 +38,22 @@ set(CBLAS1 scabs1.f scasum.f scnrm2.f90 icamax.f caxpy.f ccopy.f
 set(DBLAS1 idamax.f dasum.f daxpy.f dcopy.f ddot.f dnrm2.f90
 	drot.f drotg.f90 dscal.f dsdot.f dswap.f drotmg.f drotm.f)
 
+set(DB1AUX sscal.f isamax.f)
```

Comment:
This is the first time I am checking which routines use the auxiliary routines in `*B1AUX`. I couldn't find which double precision routines use `ISAMAX`. Would you please be able to tell? Thanks.

Moreover, I noticed that some of the dependencies, like `SASUM` in `CB1AUX`, are only needed for testing routines. Maybe we should refactor it.

Reply:
> I couldn't find which double precision routines use ISAMAX. Would you please be able to tell? Thanks.

The dependency chain is `dsgesv` which depends on `sgetrf`, `sgetrf2`, `isamax`.

> Moreover, I noticed that some of the dependencies, like SASUM in CB1AUX, are only needed for testing routines. Maybe we should refactor it.

When working on this issue, I was surprised to see that someone had already put in the effort to separate all the code needed for the different floating-point types. In fact, this has been part of the CMake setup since its introduction in 2009. All that was left to do was to update the lists. Since there are no automated tests in place for selective builds, such a change would be prone to be broken again in the future.